build.sh

```diff
@@ -10,3 +10,9 @@ for path in test/Serilog.Tests/project.json; do
     dotnet build ${dirname} -f netcoreapp1.0 -c Release
     dotnet test ${dirname} -f netcoreapp1.0  -c Release
 done
+
+for path in test/Serilog.PerformanceTests/project.json; do
+    dirname="$(dirname "${path}")"
+    dotnet build ${dirname} -f netcoreapp1.0 -c Release
+    dotnet test ${dirname} -f netcoreapp1.0  -c Release
```

Comment:
For consistency we can turn it off however the travis build runs in 8 mins & 19mins ATM due to the reduced numbers of targets.

* `Elapsed time 8 min 9 sec`
* `Elapsed time 19 min 1 sec`

I will move to a new `.sh` and `.ps1` to run the perf tests in isolation.